genienlp/data_utils/bootleg.py

```diff
@@ -178,19 +187,45 @@ def disambiguate_mentions(self, config_args):
 
     def post_process_bootleg_types(self, title):
         types = None
-        for key in self.wiki2normalized_type.keys():
-            if fnmatch.fnmatch(title, key):
-                types = self.wiki2normalized_type[key]
+        for pair in self.wiki2normalized_type:
+            if fnmatch.fnmatch(title, pair[0]):
+                types = pair[1]
 
         typeqids = None
         if types is not None:
             if isinstance(types, str):
-                typeqids = [self.type_vocab_to_entityqid[types]]
+                typeqids = [self.type_vocab_to_typeqid[types]]
             elif isinstance(types, (list, tuple)):
-                typeqids = [self.type_vocab_to_entityqid[type_] for type_ in types]
+                typeqids = [self.type_vocab_to_typeqid[type_] for type_ in types]
 
         return typeqids
-
+    
+    def textualize_features(self, feat):
```

Comment:
Maybe "features_to_text" ? Or better, "entities_to_text", cause we're moving away from the generic word "feature"